tests/bazel/Makefile

```diff
@@ -0,0 +1,14 @@
+include $(dir $(MAKEFILE_LIST))../common.mk
+
+all:
+	bazel build --force_pic --collect_code_coverage=True //main:testcase
```

Comment:
gcda files aren't created until the tests are run, usually with the `coverage` subcommand.

It seems that you can find them in `bazel-testlogs`:
https://github.com/bazelbuild/bazel/issues/20041#issuecomment-1792441015

Reply:
although from reading this script, it seems like `.gcov` files are created:
https://github.com/bazelbuild/bazel/blob/master/tools/test/collect_cc_coverage.sh

Reply:
Will check with this command instead of executing directly the test case.